lib/DDGC/DB/Result/GitHub/Comment.pm

```diff
@@ -0,0 +1,42 @@
+package DDGC::DB::Result::GitHub::Comment;
+# ABSTRACT:
+
+use Moose;
+use MooseX::NonMoose;
+extends 'DDGC::DB::Base::Result';
+use DBIx::Class::Candy;
+use namespace::autoclean;
+
+table 'github_comment';
+
+primary_column id          => {data_type => 'bigint', is_auto_increment => 1};
+unique_column  github_id   => {data_type => 'bigint', is_nullable => 0};
+column github_repo_id      => {data_type => 'bigint', is_nullable => 0};
+column github_user_id      => {data_type => 'bigint', is_nullable => 0};
+column number              => {data_type => 'bigint', is_nullable => 0};
+column body                => {data_type => 'text',   is_nullable => 0};
+column created_at          => {data_type => 'timestamp without time zone', is_nullable => 0};
+column updated_at          => {data_type => 'timestamp without time zone', is_nullable => 0};
+column gh_data             => {
+    data_type        => 'text',
+    is_nullable      => 0,
+    serializer_class => 'AnyJSON',
+    default_value    => '{}',
```

Comment:
@kablamo does this prevent errors when calling from_json and to_json in case the string is empty? Just guessing

Also, what about the AnyJSON serializer class? What does it do? I think we use the JSON serializer class elsewhere...